microsoft-edge/extensions-chromium/publish/contact-extensions-team.md

```diff
@@ -28,4 +28,4 @@ You can contact the Microsoft Edge extensions support team request help, submit
 <!-- ====================================================================== -->
 ## Report copyright or trademark infringement
 
-If you think an item in the Microsoft Edge Add-ons website violates a copyright or trademark, please fill out the [Reporting Infringement](https://www.microsoft.com/en-us/concern/dmca) form. While filling up the Step-3 of the form in the **Product/Service/App** option, ensure you select **‘Microsoft Store on Windows’**. The Microsoft Edge extensions team will review your report and then take the necessary action.
\ No newline at end of file
+If you think an item in the Microsoft Edge Add-ons website violates a copyright or trademark, please complete the [Reporting Infringement](https://www.microsoft.com/en-us/concern/dmca) form. When finishing Step-3 of the form in the **Product/Service/App** option, ensure you select **'Microsoft Store on Windows'**. The Microsoft Edge extensions team will review your report and then take the necessary action. 
```

Comment:
'please' has 37 hits in 27 files.  I think it fits here b/c we're inviting user to go out of their way and do a favor.
https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/p/please - "Avoid please except in situations where the customer is asked to do something inconvenient or the application or site is to blame for the situation."
```suggestion
If you think an item in the Microsoft Edge Add-ons website violates a copyright or trademark, please complete the [Reporting Infringement](https://www.microsoft.com/en-us/concern/dmca) form. When finishing Step 3 of the form in the **Product/Service/App** option, make sure to select **'Microsoft Store on Windows'**. The Microsoft Edge extensions team will review your report and then take the necessary action.
```